code/modules/clothing/rogueclothes/neck/neck.dm

```diff
@@ -440,7 +440,7 @@
 /obj/item/clothing/neck/roguetown/fencerguard
 	name = "fencer neckguard"
 	icon_state = "fencercollar"
-	armor = list("blunt" = 90, "slash" = 100, "stab" = 80, "piercing" = 100, "fire" = 0, "acid" = 0)
+	armor = list("blunt" = 90, "slash" = 100, "stab" = 80, "bullet" = 100, "fire" = 0, "acid" = 0)
```

Comment:
as with the other fencer items I would increase stab protection and drop slash/blunt a bit.